countrycodepicker/src/main/java/com/rilixtech/widget/countrycodepicker/CountryCodeArrayAdapter.java

```diff
@@ -66,14 +67,13 @@ private void setData(Country country, ViewHolder viewHolder) {
     viewHolder.tvCode.setVisibility(View.VISIBLE);
     viewHolder.llyFlagHolder.setVisibility(View.VISIBLE);
     Context ctx = viewHolder.tvName.getContext();
-    String name = country.getName();
```

Comment:
I think we should use this as a fallback in case the Locale is null (invalid `iso` maybe?) so at least we show something

Reply:
We should probably also remove the translation strings for other languages (various `strings_country.xml` files in res/values-xx)

Reply:
Yes we can remove this, but i think it is safe for future reference.